builtin/game/chatcommands.lua

```diff
@@ -677,6 +677,38 @@ core.register_chatcommand("spawnentity", {
 	end,
 })
 
+core.register_chatcommand("setnode", {
+	params = "<NodeName> [<X>,<Y>,<Z>]",
+	description = "Places node at given (or your) position",
+	privs = {interact = true, server = true},
```

Comment:
If 'server' is a required priv, is 'interact' redundant and unnecessary?

Reply:
If 'interact' is enough to use this command, i think it is wrong that the average player can place nodes absolutely anywhere on the map. Only being able to place nodes where a player can reach is an important safeguard.
I think only those with 'server' priv should be able to use this command due to its power.

Reply:
Privs are always AND. Chat commands use `core.check_player_privs`, which as you might already know, returns a boolean and a list of missing privs, and the first value is true only if all the required privs are satisfied.

Reply:
Thanks, 'interact' seems unnecessary then.